mantle/rbx_api/src/groups/models.rs

```diff
@@ -0,0 +1,20 @@
+use serde::Deserialize;
+
+use crate::models::GroupId;
+
+#[derive(Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct GroupAllRolesResponse {
+    pub group_id: GroupId,
+    pub roles: Vec<GroupRoleResponse>,
+}
+
+#[derive(Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct GroupRoleResponse {
+    pub id: u64,
```

Comment:
Are these typically small or large numbers? If large I think I'd switch this to an `AssetId`, otherwise it's fine

Reply:
They're large, but I think AssetId would be a bit of a misnomer here?